website/docs/docs/collaborate/git/setup-azure.md

```diff
@@ -114,8 +114,13 @@ A service user account must have the following Azure DevOps permissions for all
 
 Some of these permissions are only accessible via the Azure DevOps API, for which documentation can be found [here](https://docs.microsoft.com/en-us/azure/devops/organizations/security/namespace-reference?view=azure-devops). We’ve also detailed more information on Azure DevOps API usage below to help accelerate the set up. Alternatively, you can use the Azure DevOps UI to enable permissions, but you cannot get the least permissioned set.
 
-:::info  Provide the service user with required permissions before setting up a dbt Cloud project
-This service user's permissions will also power which repositories a team can select from during dbt project set up, so an Azure DevOps admin must grant at minimum Project Reader access to the service user before setting up a project in dbt Cloud.
+:::info  Provide the service user with required permissions before creating a new dbt Cloud project
+This service user's permissions will also power which repositories a team can select from during dbt project set up, so an Azure DevOps admin must grant at minimum Project Reader access to the service user before creating a new project in dbt Cloud. If you are migrating an existing dbt project to use the native Azure DevOps integration, the dbt Cloud account's service user must have proper permissions on the repository before migration.
+:::
+
+:::info  Turn off multi-factor authentication for the service user
+While it's common to enforce multi-factor authentication for normal user accounts, service user authentication must not require an additional factor. If the service user is beholden to a 2nd factor, this will cause an interrupt for production runs causing a failure to clone occasionally. It's best practice to remove any additional burden of proof of identity for service users, in order for the OAuth access token to sufficiently work.
```

Comment:
can i confirm if this a failure to clone repo step right? 

> production runs causing a failure to clone occasionally.

Reply:
yep- that's right!